### ISSUE TITLE
[BUG]: potential bug to empty decks

### ISSUE THREAD

**Issue by gustavosmendes1** [ankidroid/Anki-Android#14669]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

Clicked on numbers of deck
![Screenshot_20231105_123651_AnkiDroid](https://github.com/ankidroid/Anki-Android/assets/105264313/504220d3-abfd-4917-a220-df372491e959)

### Expected behaviour

Start flashcards

### Actual behaviour

Showing empty flashcards
![Screenshot_20231105_123544_AnkiDroid](https://github.com/ankidroid/Anki-Android/assets/105264313/0fdd3c9c-b070-4c95-b503-d670e4ddf931)

### Debug info

```text
AnkiDroid Version = 2.17alpha3 (42a8c8c55c200f372b0bd0b462e4b58554b72fe5)

Android Version = 13

ProductFlavor = play

Manufacturer = samsung

Model = SM-A546E

Hardware = s5e8835

Webview User Agent = Mozilla/5.0 (Linux; Android 13; SM-A546E Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/118.0.0.0 Mobile Safari/537.36

ACRA UUID = b10a232d-8718-4314-92f9-d47e5dcecbaf

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by gustavosmendes1** [comment]
The subtitle says that the deck is empty

**Comment by lukstbit** [comment]
Duplicate of #14538 
I made a fix for this which should be available from 2.17alpha4.
